### PR TITLE
fix: differentiate between USDC and USDC.e

### PR DESCRIPTION
**Pull request by H3xept**
https://lifi.atlassian.net/browse/LF-3261

### CODE REVIEW

File: src/coins.ts [hunk 1]
```diff
@@ -847,18 +847,16 @@ const basicCoins: BasicCoin[] = [
...
       [ChainId.ONE]: {
         address: '0x985458e523db3d53125813ed68c274899e9dfab4',
         decimals: 6,
       },
       [ChainId.AVA]: {
-        address: '0xa7d7079b0fead91f3e65f86e8915cb59c1a4c664',
+        address: '0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e',
         decimals: 6,
-        symbol: 'USDC.e',
-        name: 'USD Coin',
```
Comment (maxklenk) [inline src/coins.ts:861]: Currently the symbol and name of all the USDC coins will be `USDC`. Should we instead set the name to `USD Coin` for all USDC token?

**Review by maxklenk — CHANGES_REQUESTED** [review]

**Comment by maxklenk** [comment]
> I also differentiated between USDC and USDC.e on avalanche: https://www.circle.com/blog/understanding-avalanche-usdc-vs.-usdc.e @maxklenk can you check if this makes sense?

Yes, it is the same case on Avalanche and will be on more chains as soon as Circle releases there. This PR is a breaking change, so we should make a major release of the types.

File: src/coins.ts (hunk 1 above)
Comment (H3xept) [reply src/coins.ts:861]: Yep

**Review by maxklenk — APPROVED** [review]
Great

**Pull request merged**